### DISCUSSION TITLE
如何获取机器人接收到的信息判断是否被@然后针对回复

### DISCUSSION THREAD

**Discussion by jackatls** [GraiaProject/Application#116] [Q&A]
@bcc.receiver("GroupMessage")
async def group_message_listener(app: GraiaMiraiApplication, group: Group):
    res = await app.fetchMessage()
    print(res)
我尝试这么做，但结果一直为空

**Comment by GreyElaina** [comment]
建议去看看其他人写的。

**Reply by jackatls** [reply]
嗯，看了点别人写的和函数的注释，现在基本功能都能写了，主要都是message.get
